auth/provider_auth.go

```diff
@@ -0,0 +1,62 @@
+// wire_peer_auth.go - client wire authentication for peers
+// Copyright (C) 2017  David Anthony Stainton
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+// Package provides mixnet Provider authentication
+package auth
+
+import (
+	"crypto/subtle"
+
+	"github.com/katzenpost/client/config"
```

Comment:
referenced file doesn't exist?

Reply:
ok i've included that code in this branch